src/app/main/api-gest-console/modules/aig-standard/components/cpv-new-update-dialog/cpv-new-update-dialog.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, OnInit } from '@angular/core';
+import { MatDialogRef } from '@angular/material/dialog';
+
+@Component({
+    templateUrl: './cpv-new-update-dialog.component.html',
+    styleUrls: ['./cpv-new-update-dialog.component.scss']
+})
+export class AigCpvNewUpdateDialogComponent implements OnInit {
+    constructor(
+        public matDialogRef: MatDialogRef<AigCpvNewUpdateDialogComponent>,
+    ) { }
```

Comment:
add this in the contructor parameters for take data info from dialog launcher:
`@Inject(MAT_DIALOG_DATA) public data: any`